include/openssl/whrlpool.h

```diff
@@ -46,9 +46,9 @@ void WHIRLPOOL_BitUpdate(WHIRLPOOL_CTX *c, const void *inp, size_t bits);
 int WHIRLPOOL_Final(unsigned char *md, WHIRLPOOL_CTX *c);
 unsigned char *WHIRLPOOL(const void *inp, size_t bytes, unsigned char *md);
 
-# ifdef __cplusplus
+#  ifdef __cplusplus
 }
-# endif
+#  endif
```

Comment:
This is wrong. Please use `DEPRECATEDIN_3_0` with the function signatures and only use the `OPENSSL_NO_DEPRECATED_3_0` guard around stuff that aren't function signatures.

Reply:
This uglifies things a bit, but done.

Reply:
And the build errors tell me why :)